cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -288,14 +293,41 @@ func (t *UnmanagedCluster) Deploy(scConfig *config.UnmanagedClusterConfig) (int,
 		}
 	}
 
-	// 8. Update kubeconfig and context
+	// 8. Install profile if specified
+	if t.config.Profile != "" {
+		log.Eventf(logger.GlobeEmoji, "Installing Profile %s\n", t.config.Profile)
+
+		t.profilePkg, err = resolveProfilePkg(pkgClient, t.config.Profile, t.config.ProfileVersion)
+		if err != nil {
+			return ErrProfileInstall, fmt.Errorf("could not resolve profile. Error: %s", err.Error())
+		}
+
+		if t.config.ProfileVersion == "" {
+			log.Style(outputIndent, color.FgYellow).Warnf("Installing profile without version specified. Using version %s\n", t.profilePkg.pkgVersion)
+		} else {
+			log.Style(outputIndent, color.Faint).Infof("Using profile version %s\n", t.config.ProfileVersion)
+		}
+
+		if t.config.ProfileConfig == "" {
+			log.Style(outputIndent, color.FgYellow).Warnf("Installing profile with no configuration file\n")
```

Comment:
Should this be a warning?

I wonder if it's expected and OK for some future metapackages to just install without config?

Reply:
Hmmmm good question.

My gut instinct is that _most_ packages will need at least some configuration and installing the default package without any values "works" ok but we should log a warning to the user that the package is using all default values and may not work 100% out of the box.

Even something [simple like `fluent-bit` needs](https://github.com/vmware-tanzu/community-edition/tree/main/addons/packages/fluent-bit/1.7.5) annotations configurations and stream sources to forward logs successfully.

---

Up to you, I don't know enough about the packaging expectations to make a call on this one

Reply:
Yeah, I hear you.

I'd hope most packages that could be used as profiles would have "sensible no-config defaults".

Either way we're guessing.

Let's keep the warning and see if it raising improper concern for users over time. :+1: 

_resolving_